tailormap-components/projects/core/src/lib/user-interface/attributelist/attributelist-panel/attributelist-panel.component.html

```diff
@@ -33,7 +33,7 @@
   <ng-template #withLayers>
     <mat-tab-group #tabgroup animationDuration="0ms"
                    (selectedTabChange)="onSelectedTabChange();">
-      <mat-tab *ngFor="let layer of layers; let index = index" [label]="layer.name">
+      <mat-tab *ngFor="let layer of layers; let index = index; trackBy: trackByLayerId" [label]="layer.name">
```

Comment:
trackBy improves the performance a lot since Angular will not update once the result of this trackByLayerId method is equal

Reply:
Nice!